packages/mobile/src/home/HomeTokenBalance.tsx

```diff
@@ -73,15 +74,12 @@ function useErrorMessageWithRefresh() {
   const { t } = useTranslation()
 
   const tokensInfoUnavailable = useSelector(tokensInfoUnavailableSelector)
-  const tokenFetchLoading = useSelector(tokenFetchLoadingSelector)
-  const lastSuccessfulFetch = useSelector(lastTokenFetchSelector)
+  const tokenFetchError = useSelector(tokenFetchErrorSelector)
+  const localCurrencyError = useSelector(localCurrencyExchangeRateErrorSelector)
 
   const dispatch = useDispatch()
 
-  const shouldShowError =
-    (tokensInfoUnavailable ||
-      lastSuccessfulFetch < Date.now() - TIME_UNTIL_TOKEN_INFO_BECOMES_STALE) &&
```

Comment:
Now that we remove this condition, how do we measure the time from the last successful fetch?
Is that logic somehow considered in the `tokensInfoUnavailable`?

Reply:
It is! It's in the tokens reducer, I think I didn't take into account when I first implemented it which is why I added the other check but it was redundant